ddocs/Readme

```diff
@@ -0,0 +1,18 @@
+
+CouchDB design docs for medic-api
+
+The modules here are CouchDB compatible (ships with Spidermonkey 1.8.5) but
+tested and linted with NodeJS.
+
+More info:
+
+  http://docs.couchdb.org/en/2.0.0/install/unix.html
+  https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Releases/1.8.5
+ 
+To build the design doc run `make`.
+
+To install it on a running CouchDB run `make install`.
+
+Compiled design docs (JSON files) are committed to the repository and
+installed by medic-api during startup.
+
```

Comment:
Put this readme in the root readme in its own section so it'll be easier to find.